entrypoint.sh

```diff
@@ -44,6 +44,7 @@ elif [ -n "$POSTGRES_PORT_5432_TCP" ]; then
 		echo >&2 "  Connecting to DB_HOST ($DB_HOST)"
 		echo >&2 "  instead of the linked postgres container."
 	fi
+	: ${DB_NAME:='postgres'}
```

Comment:
i still don't get the point why using another database-technology should alter a default.